src/status_im2/navigation/core.cljs

```diff
@@ -180,7 +180,10 @@
                                  opts)}})))
 
 ;; toast
-(navigation/register-component "toasts" (fn [] views/toasts) js/undefined)
+(navigation/register-component "toasts"
+                               #(gesture/gesture-handler-root-hoc views/toasts
+                                                                  #js {:flex 0})
+                               (fn [] views/toasts))
```

Comment:
here `:flex 0` is the same as `flex: 0 0 auto` in CSS.
We need this to override the HOC default layour which is `flex 1`.

If we don't override this property, then the toast layout will catch all taps/gestures, so no other action would be available until the toast disappears

Reply:
hmm, this seems like it might be worth adding a comment in the code? WDYT?

Reply:
Yes! I will add it, I think it's worth it